packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-suggestions.tsx

```diff
@@ -0,0 +1,258 @@
+import {Component, h, Prop, State, Element} from '@stencil/core';
+import {
+  InitializableComponent,
+  InitializeBindings,
+  Bindings,
+  BindStateToController,
+} from '../../utils/initialization-utils';
+import ArrowRight from '../../images/arrow-right.svg';
+import ArrowDown from '../../images/arrow-down.svg';
+import {
+  buildInteractiveResult,
+  buildSmartSnippetQuestionsList,
+  SmartSnippetQuestionsList,
+  SmartSnippetQuestionsListState,
+  SmartSnippetRelatedQuestion,
+} from '@coveo/headless';
+import {Hidden} from '../common/hidden';
+import {Heading} from '../common/heading';
+import {LinkWithResultAnalytics} from '../result-link/result-link';
+import {Button} from '../common/button';
+import {randomID} from '../../utils/utils';
+
+/**
+ * The `atomic-smart-snippet-suggestions-suggestions` component displays an accordion of questions related to the query with their corresponding answers.
+ *
+ * You can style the snippets by inserting a template element like this:
+ * ```html
+ * <atomic-smart-snippet-suggestions>
```

Comment:
what is the difference between this approach and using `snippetStyle`?

Reply:
I just saw the code above, and I'd like to rephrase the question - why are we allowing users to do both?

Reply:
This was copied from SmartSnippet, which works the same way.

The first method (with a template element) is easier to define from a static/vanilla page.

The attribute is easier to define from JSX, and may be the only option in some frameworks.